src/routing/Router.tsx

```diff
@@ -1,38 +1,66 @@
 import React from 'react';
-import { BrowserRouter, Routes } from 'react-router-dom';
+import {
+  BrowserRouter,
+  createRoutesFromChildren,
+  RouteObject,
+  Routes,
+} from 'react-router-dom';
 import { Formik } from 'formik';
 import { RouterProps } from './types';
 
 import FormTitle from '../form-layout/FormTitle';
 import FormFooter from '../form-layout/FormFooter';
+import { RouterContext } from './RouterContext';
+
+const routeObjectsReducer = (routeObjectsArray: RouteObject[]) => {
+  return routeObjectsArray.reduce<string[]>(
+    (accumulator, current): string[] => {
+      if (current.children) {
+        return [
```

Comment:
is there any way we can test the logic for this function?

Reply:
Yeah plus 1 for adding tests around this 👍